src/D2L.Bmx/Okta/OktaClient.cs

```diff
@@ -51,10 +52,8 @@ IOktaAuthenticatedClient IOktaClientFactory.CreateAuthenticatedClient( string or
 		return new OktaAuthenticatedClient( httpClient );
 	}
 
-	private static Uri GetBaseAddress( string org ) {
-		return org.Contains( '.' )
-			? new Uri( $"https://{org}/api/v1/" )
-			: new Uri( $"https://{org}.okta.com/api/v1/" );
+	private static Uri GetApiBaseAddress( string org ) {
+		return new Uri( $"{org}api/v1/" );
```

Comment:
Would prefer a safer way to concat URL parts.
Here you're implicitly requiring that the `org` passed in ends in `/`